### PR TITLE
Fix questions in web/html/html-structure/html-structure-define.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Remove the images from the Practice and Revision questions and add text-only questions

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/web/html/html-structure/html-structure-define.md?ref=13efc027bfe59cbbec0ecafb6e9895828cefb9f4

File: web/html/html-structure/html-structure-define.md [hunk 1]
```diff
@@ -28,52 +28,55 @@ The typical structure of HTML elements are defined as:
...
+Enki: ???
+```
 
 * element
 * opening tag
 * end tag
 * attribute
-* attribute's name
-* attribute's value
+* attributes name
```
Comment (nem035) [inline web/html/html-structure/html-structure-define.md]: Wasn't the original text correct? It's "name of an attribute" or "attribute's name", no?

File: web/html/html-structure/html-structure-define.md [hunk 2]
```diff
@@ -28,52 +28,55 @@ The typical structure of HTML elements are defined as:
...
-* attribute's value
-* content
-* value
-* rule
+element: ???
+opening tag: ???
+end tag: ???
+attribute: ???
+attribute's name: ???
+attributes value: ???
```
Comment (nem035) [inline web/html/html-structure/html-structure-define.md]: Missing apostrophe? Should be "attribute's" no?

Comment (Stefan-Stojanovic) [reply web/html/html-structure/html-structure-define.md]: Yes, I overlooked that grammar mistake

**Pull request merged**